keda/templates/keda-vpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.verticalPodAutoscaler.keda.enabled }}
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: {{ .Values.operator.name }}-vpa
+  namespace: {{ .Release.Namespace }}
```

Comment:
Any reason why you are not adding the annotations ?

Reply:
Othereise looks good to me

Reply:
No particular reason, I just didn't thought they are necessary :shrug: 
I can add it if you prefer though